button-examples/Web/web-button-example.md

```diff
@@ -111,20 +114,19 @@ Add an icon to your `mdc-button` instance using the following steps:
 
 ### Sass mixins
 
-Use Sass mixins when you want to customize the look and feel of your buttons. Go to [sass-lang.com](https://sass-lang.com/install) for installation instructions.
-
-Buttons support [Material Theming](https://material.io/components/buttons/#theming) and can be customized in terms of color, typography and shape.
+Buttons support [Material Theming](https://material.io/components/buttons/#theming) and can be customized in terms of color, typography and shape. Material uses Sass to customize the look and your components. Go to [sass-lang.com](https://sass-lang.com/install) for installation instructions.
 
 <details><summary><b>Expand for instructions to use Sass mixins to customize your <code>mdc-button</code> or <code>mdc-icon-button</code></b></summary>
 
 Before using Sass mixins for your project you will need to do the following:
 
-* Add the Sass package to your `*.json file` under `devDependencies`:
+* Add the Sass package to your `*.json file` under `devDependencies` or `npm install sass --save` to install it in your terminal:
 ```json
 "devDependencies": {
   "sass": "^1.24.3"
 }
 ```
+// Do we need to do this bit? I think it's highly dependant on a user's environment (the sassrc file). I haven't done this in a long time and used Sass with a different build step instead
```

Comment:
That I'm not sure about.

@abhiomkar Do you know if this step applies across user environments?